### DISCUSSION TITLE
Envisioning the Future of OWASP: A Proposal for New Key Roles

### DISCUSSION THREAD

**Discussion by ghost** [OWASP/www-committee-project#69] [General]
Hello OWASP Community,

As an active member of our community, I want to share a thoughtful proposition, originally put forth by a fellow community member, concerning the creation of two new roles: a Community Manager and a Projects Manager. I believe these additions could be game-changing for our organization, and I'd like to add a third role to this discussion, a Grant Coordinator, for which I volunteer initially with the vision of it becoming a full-time position in due time.

The **Community Manager**, as proposed, would ideally combine expertise in Application Security, proficiency in social media management, and a strong marketing background. The primary responsibility of this role would be acting as a champion for our organization, driving the promotion of our projects and chapters on various platforms.

This role would involve:

1. Managing the OWASP Twitter account, blog, and other related channels.
2. Harnessing the #amplify channel to promote content and foster engagement.
3. Spearheading and contributing to discussions on Slack to boost active participation.
4. Showcasing the accomplishments of the board and staff for enhanced visibility.
5. Identifying potential marketing opportunities by leveraging a thorough understanding of OWASP's project portfolio.

Additionally, a **Projects Manager** role was suggested. This role would require a degree of technical knowledge and involve overseeing ongoing projects while maintaining the overall health of our project portfolio.

The responsibilities of this role would include:

1. Streamlining the portfolio by removing inactive or abandoned projects.
2. Addressing project demotions and inspiring continuous quality representation of OWASP.
3. Monitoring and addressing discrepancies in project activity levels throughout the year.
4. Ensuring efficient coordination with the Project Committee, primarily composed of busy project leaders.

Along with these two roles, I propose the addition of a **Grant Coordinator**. While I am willing to volunteer for this role initially, the goal is to evolve this into a permanent, full-time position. This role is crucial for obtaining funding and ensuring the financial sustainability of our projects.

Responsibilities of the Grant Coordinator would entail:

1. Building and maintaining relationships with potential donors and sponsors.
2. Exploring innovative fundraising strategies to supplement traditional methods.
3. Advocating for the financial needs of our projects, ensuring efficient allocation of resources.

Let's have a discussion about these proposed roles and their potential impact on our organization's future. I believe our collective thoughts and feedback would provide invaluable insights for OWASP's continued growth and success. What are your thoughts on these proposed roles? How do you envision these positions contributing to OWASP's development?

Looking forward to hearing your perspectives!

Best,
Donnie

**Comment by psiinon** [comment]
I think these roles are long overdue!
I'd also argue that the Grant Coordinator is the first one that should be filled...

**Comment by tghosth** [comment]
Sounds great !

**Comment by kingthorin** [comment]
It occurred to me this morning while scrolling on Twitter that with the "Marketing" bit mentioned for the Community Manager we should probably be actively 'advertising' OWASP's projects. It'd be wonderful to encounter banner etc for OWASP stuff while browsing. (Not that I like ads, but still...)
